src/frontend/e2e/utils/sample.ts

```diff
@@ -52,6 +52,7 @@ export class SampleUtil {
           defaults?.sample?.public_identifier,
           GeneralUtil.generatePublicSampleId()
         ) as string,
+        status: ""
```

Comment:
Any reason we want the status to be blank?

Reply:
since i changed to complete/failed i dont need any specific content on it, but i have set it as complete now

Reply:
hmm. I am not sure why we are setting status to a function that generates for upload data. status is part of response, not upload so I am curious about this change